app/main.py

```diff
@@ -1,3 +1,57 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float, Distance]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[int, float]) -> Distance:
+        if type(other) == int or type(other) == float:
+            return Distance(
+                km=self.km + other
+            )
+        return Distance(
+            km=self.km + other.km
+        )
+
+    def __iadd__(self, other: Union[int, float, Distance]) -> Distance:
+        if type(other) == int or type(other) == float:
+            self.km += other
+        else:
+            self.km += other.km
+        return self
+
+    def __mul__(self, other: Union[int, float]) -> Distance:
+        return Distance(
+            km=self.km * other
+        )
+
+    def __truediv__(self, other: Union[int, float]) -> Distance:
+        return Distance(
+            km=round(self.km / other, 2)
+        )
+
+    def __lt__(self, other: Union[int, float]) -> bool:
+        return self.km < other
```

Comment:
other also can be the Distance instance

Reply:
so you need to implement aditional logick that will work if other is Distance instanse.